MerlinAU.sh

```diff
@@ -534,13 +534,18 @@ _CheckForNewScriptUpdates_()
    rm -f "$SCRIPTVERPATH"
 
    # Download the latest version file from the source repository
-   curl -LSs --retry 4 --retry-delay 5 "${SCRIPT_URL_BASE}/version.txt" -o "$SCRIPTVERPATH"
+   wget --quiet --tries=4 --waitretry=5 "${SCRIPT_URL_BASE}/version.txt" -O "$SCRIPTVERPATH"
 
    if [ $? -ne 0 ] || [ ! -s "$SCRIPTVERPATH" ]
    then scriptUpdateNotify=0 ; return 1 ; fi
 
    # Read in its contents for the current version file
    DLRepoVersion="$(cat "$SCRIPTVERPATH")"
+   if [ -z "$DLRepoVersion" ]; then
+       echo "Variable for downloaded version is empty."
+       UpdateNotify=0
```

Comment:
BTW, the variable name was changed to "`scriptUpdateNotify`" to make it distinct & clear that it's not related to the F/W Update notification.